GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -31,22 +31,31 @@
 // Key constants used for encode and decode.
 static NSString *const kAuthenticationKey = @"authentication";
 static NSString *const kGrantedScopesKey = @"grantedScopes";
-static NSString *const kUserIDKey = @"userID";
-static NSString *const kServerAuthCodeKey = @"serverAuthCode";
 static NSString *const kProfileDataKey = @"profileData";
-static NSString *const kHostedDomainKey = @"hostedDomain";
+static NSString *const kAuthState = @"authState";
+static NSString *const kIDToken = @"idToken";
 
 // Parameters for the token exchange endpoint.
 static NSString *const kAudienceParameter = @"audience";
 static NSString *const kOpenIDRealmParameter = @"openid.realm";
 
 
-@implementation GIDGoogleUser
+@implementation GIDGoogleUser {
+  OIDAuthState *_authState;
+  NSString *_idToken;
+}
+
+@synthesize serverAuthCode = _serverAuthCode;
```

Comment:
Do we need to synthesize these if they're already declared as properties in the public header?

Reply:
Yes. These are read-only properties and we implemented the getter so they are not auto synthesized.

Reply:
Are you using the ivars that are being synthesized here?

Reply:
Good catch. These are not used any more.